server/app/lib/actions/fusor/deployment/cloud_forms/run_appliance_console.rb

```diff
@@ -77,7 +77,7 @@ def run
           def run_command(cmd)
             ::Fusor.log.info "Running: #{cmd}"
             status, output = Utils::Fusor::CommandUtils.run_command(cmd)
-            ::Fusor.log.debug "Status: #{status}, output: #{output}"
+            ::Fusor.log.info "Status: #{status}, output: #{output}"
```

Comment:
Not sure this is useful at INFO. We really only care if it failed and that is handled in the run method when we fail the job:

```
status, output = run_command(cmd)
fail _("Unable to run appliance console: %{output}") % { :output => output.join('; ') } unless status == 0
```